cppcache/integration/test/Position.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "Position.hpp"
+
+#include <wchar.h>
+
+#include <cwchar>
+
+#include <geode/DataInput.hpp>
+#include <geode/DataOutput.hpp>
+
+namespace DataSerializableTest {
+
+int32_t Position::cnt = 0;
+
+Position::Position() { init(); }
+
+Position::Position(const char* id, int32_t out) {
```

Comment:
This source smells like it was lifted from somewhere without care to bring it up to our high standards. Can you please:
* Use `std::string` rather than `char *`.
* Readable and descriptive variable names. (What is `out`?)
* Inner class members should not be `CacheableX` types.
* Don't use C methods where C++ methods exist, like `sprintf`.

Reply:
You're right. This was just copied from the testobject folder, which is used by the old legacy framework tests. I agree to include it in the new framework tests it needs to be sanitized.